### DISCUSSION TITLE
Extend zigbee range

### DISCUSSION THREAD

**Discussion by chgaertner** [Koenkk/zigbee2mqtt#23381] [General]
Dear all,

sorry, if this is a dumb question, but I did not find an answer.

If I want to extend the range of my zigbee network, I can use (a) any mains powered device or (b) a zigbee stick flashed with the router firmware.

Is it enough to put the stick with the router firmware in an usb power supply or does it have to be a computer? Or, to say it this way: does an usb router zigbee stick need more than the power on the respective usb pins?

Greetings, Chris

**Comment by lux73** [comment]
USB Power supply is completely sufficient - but you have to connect the Router to your Zigbee Network also

**Reply by chgaertner** [reply]
Thank you, I will give it a try with the SONOFF ZBDongle-E based router as described on this webpage.

Greetings, Chris

**Reply by lux73** [reply]
no really the best Option in my opinion. Dongle E are marked as experimental

Read the Forums. The are several Threads about Zigbee Network stability with Dongle E

**Reply by chgaertner** [reply]
I read this page (https://www.zigbee2mqtt.io/devices/ZBDongle-E.html) and the following statement:

_"You can create a powerful Zigbee router from an ordinary Sonoff Zigbee 3.0 USB Dongle Plus (Model "ZBDongle-E" with EFR32MG21 chip) by flashing a router firmware onto the device."_

I will check the forum ...
